iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/ShuntXml.java

```diff
@@ -52,9 +62,15 @@ protected ShuntCompensator readRootElementAttributes(ShuntCompensatorAdder adder
         double bPerSection = XmlUtil.readDoubleAttribute(context.getReader(), "bPerSection");
         int maximumSectionCount = XmlUtil.readIntAttribute(context.getReader(), "maximumSectionCount");
         int currentSectionCount = XmlUtil.readIntAttribute(context.getReader(), "currentSectionCount");
+        boolean regulating = XmlUtil.readBoolAttribute(context.getReader(), "regulating");
```

Comment:
In those existing xml, this filed is not exist. 
Would it throw exception while using new code to read an old xml?

Reply:
Yes, this is why this PR must wait the merge of the evolution for versionning XIIDM, this will be in IIDM 1.1

Reply:
FYI, there is a branch about v1.1
https://github.com/powsybl/powsybl-core/tree/my_iidms